jbdevstudio/com.jboss.jbds.central.discovery.earlyaccess/plugin.xml

```diff
@@ -75,5 +65,30 @@ This content is unavailable. Enable Early Access to see it.</description>
          <overview url="http://www.jboss.org/tools"/>
       </connectorDescriptor>
 
+       <!-- JBoss Fuse Development -->
```

Comment:
Because spoofed content for the moment which will be removed when we release CR1a.